.github/workflows/ubuntu-touch.yml

```diff
@@ -92,12 +92,12 @@ jobs:
           path: build.clickable/*.click
           if-no-files-found: ignore
 
-      - name: Publish to Open Store
-        if: ${{ github.ref == 'refs/heads/master' && matrix.build_type == 'Release' && matrix.opengl == 'sdl' }}
-        env:
-          ARCH: ${{ matrix.arch }}
-          OPENSTORE_KEY: ${{ secrets.OPENSTORE_KEY }}
-        run: |
-          ~/.local/bin/clickable publish "* $(git log -1 --pretty=%B |         \
-                                         head -1)" --apikey ${OPENSTORE_KEY}   \
-                                         --arch ${ARCH}
+      #- name: Publish to Open Store
+      #  if: ${{ github.ref == 'refs/heads/master' && matrix.build_type == 'Release' && matrix.opengl == 'sdl' }}
+      #  env:
+      #    ARCH: ${{ matrix.arch }}
+      #    OPENSTORE_KEY: ${{ secrets.OPENSTORE_KEY }}
+      #  run: |
+      #    ~/.local/bin/clickable publish "* $(git log -1 --pretty=%B |         \
+      #                                   head -1)" --apikey ${OPENSTORE_KEY}   \
+      #                                   --arch ${ARCH}
```

Comment:
Is the disabling of this `Publish to Open Store` task intentional?

Reply:
Yes, because I think that only releases should be published. You can see at what state game currently is.

Reply:
I agree.